.circleci/config.yml

```diff
@@ -47,14 +47,24 @@ workflows:
   version: 2
   test-deploy:
     jobs:
-      - build-browser-sdk
+      - build-browser-sdk:
+          filters:
+            tags:
+              only: /.*/
       - hold:
           type: approval
           requires:
             - build-browser-sdk
           filters:
+            tags:
+              only: /^v.*/
```

Comment:
With this new behavior -- rather than CI always attempting to publish to NPM -- we'll need to trigger a publish to NPM by pushing a new `v*` tag to the Git repo. Is that correct?

Reply:
@ndonze-fs that's correct - once an `npm version` command has been run locally to create a new tag, hthe changes rendered buy `npm version` (updates to package files) must be merged into `main` and the git tag resulting from `npm version` must be pushed to the remote.

We can likely optimize this process in the future.